plugins/module_utils/k8s/service.py

```diff
@@ -0,0 +1,609 @@
+# Copyright: (c) 2021, Red Hat | Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
all files in module_utils are licensed as BSD license unless it is carried over from existing work or GNU GPL v3+ license work. @alinabuzachis Could you please clarify this?

Reply:
This is all based on code from common.py which is GPL.